site/content/docs/5.0/components/badge.md

```diff
@@ -33,8 +34,27 @@ Unless the context is clear (as with the "Notifications" example, where it is un
 
 {{< example >}}
 <button type="button" class="btn btn-primary">
-  Profile <span class="badge bg-secondary">9</span>
-  <span class="visually-hidden">unread messages</span>
+Profile <span class="badge bg-secondary">9</span>
+<span class="visually-hidden">unread messages</span>
+</button>
+{{< /example >}}
+
+Additionally, we can style the badge to show a notification in a much nicer way. This method is traditionally seen in mobile apps.
+
+{{< example >}}
+<button type="button" class="btn btn-primary position-relative">
+Alerts
+<span class="position-absolute badge start-100 rounded-circle border border-light translate-middle top-0 bg-danger pb-2">
+<span style="font-size: 0.8em;">10</span>
+</span>
+</button>
+{{< /example >}}
```

Comment:
I'd be in favor of adding a single example (the second): this one looks a bit off with border and not really rounded. Moreover having inline style to set font-size seems like a bad idea to me, and at least should not be suggested in our examples.

![image](https://user-images.githubusercontent.com/2890570/120322442-b5998000-c2e4-11eb-9671-3c99dd4ec2ac.png)

The second example works great and is really good :)

Reply:
Hello @ffoodd. Thank you for your suggestions, I will fix them as soon as I can.